### PR TITLE
KAA-1407: Simplify git flow

### PR DESCRIPTION
**Pull request by rasendubi**
/cc @akokhanovskyi @Slash32 @ikulikov @vadimol @forGGe

### CODE REVIEW

**Comment by Slash32** [comment]
LGTM

File: doc/Customization-guide/How-to-contribute/Git-flow/index.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ sort_idx: 10
...
+## Branches
 
-We have two main long-running branches: `master` and `develop`. `master` always points to the latest released version—**no pull requests are accepted to master**. `develop` is a main development branch; it reflects the latest development status. Usually, new changes should go there.
+The `master` branch represents latest development version of Kaa. Most changes should go there.
 
-There may be a number of short-living branches; most of them are used for specific jira tickets. However, `release-xxx` branches are special. They are pre-release branches which are used to test and stabilize everything before actual release. (Actual releases are tagged.) **Only bugfixes are accepted to release branches.**
+There are also `release-x.x` branches, which are stabilizing branches used for releasing maintenance releases. (All actual releases are tagged.)
 
-There may be develop branches for releases following after the next one; they took a form of `develop-xxx` (e.g., `develop-1.0.0`). Usually, you should avoid committing to them and use branches for specific features.
+If you want to backport a bug fix, open your pull request against appropriate release branch.
```
Comment (ohamov) [inline doc/Customization-guide/How-to-contribute/Git-flow/index.md:18]: `s/against appropriate/against the appropriate/g`

File: doc/Customization-guide/How-to-contribute/Git-flow/index.md [hunk 2]
```diff
@@ -82,14 +56,14 @@ GitHub will automatically update your pull request.
 
 ### Tests
 
-All pull requests are automatically checked with Travis and Jenkins. If any of the tests fails, fix it or describe why it can't be done.
+All pull requests are automatically checked with Travis and Jenkins. If any of the tests fails, fix it or describe why that can't be done.
```
Comment (ohamov) [inline doc/Customization-guide/How-to-contribute/Git-flow/index.md:60]: `s/that/it/`

**Review by ohamov — CHANGES_REQUESTED** [review]

File: doc/Customization-guide/How-to-contribute/Git-flow/index.md (hunk 2 above)
Comment (rasendubi) [reply doc/Customization-guide/How-to-contribute/Git-flow/index.md:60]: Don't agree. `that` refers to the process of fixing (not tests)

File: doc/Customization-guide/How-to-contribute/Git-flow/index.md (hunk 1 above)
Comment (rasendubi) [reply doc/Customization-guide/How-to-contribute/Git-flow/index.md:18]: ok

File: doc/Customization-guide/How-to-contribute/Git-flow/index.md (hunk 2 above)
Comment (ohamov) [reply doc/Customization-guide/How-to-contribute/Git-flow/index.md:60]: my fault, agree

**Comment by rasendubi** [comment]
Merged

**Pull request closed without merging**